### ISSUE TITLE
Error uploading an app file containing RxWebKit 1.0.0 to appstoreconnect

### ISSUE THREAD

**Issue by sy-hash** [RxSwiftCommunity/RxWebKit#34]
I am using RxWebKit 1.0.0 by carthage 0.33.0, Xcode 10.2.1.

Two types of errors occurred when uploading app build file of containing RxWebKit 1.0.0 to appstoreconnect.

### First:
It occurs in the validation when uploading to appstoreconnect)
```
ERROR ITMS-90206: "Invalid Bundle. The bundle at 'MyApp.app/Frameworks/RxWebKit.framework' contains disallowed file 'Frameworks'."
2019-06-10 07:14:44.225 altool[5834:27200] *** Error: Errors uploading '/Users/vagrant/deploy/MyApp.ipa': (
    "Error Domain=ITunesTransporterErrorDomain Code=-18000 \"ERROR ITMS-90206: \"Invalid Bundle. The bundle at 'MyApp.app/Frameworks/RxWebKit.framework' contains disallowed file 'Frameworks'.\"\" UserInfo={NSLocalizedRecoverySuggestion=ERROR ITMS-90206: \"Invalid Bundle. The bundle at 'MyApp.app/Frameworks/RxWebKit.framework' contains disallowed file 'Frameworks'.\", NSLocalizedDescription=ERROR ITMS-90206: \"Invalid Bundle. The bundle at 'Hagoita.app/Frameworks/RxWebKit.framework' contains disallowed file 'Frameworks'.\", NSLocalizedFailureReason=ERROR ITMS-90206: \"Invalid Bundle. The bundle at 'MyApp.app/Frameworks/RxWebKit.framework' contains disallowed file 'Frameworks'.\"}"
)
```

I solved the problem after removed below build option for `Target: RxWebKit`
```
ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
```

I referred to https://stackoverflow.com/a/35751497

### Second: 
It occurs after upload to appstoreconnect (received via email).
```
We identified one or more issues with a recent delivery for your app, "MyApp" 1.0.4 (1). Please correct the following issues, then upload again.

ITMS-90511: CFBundleIdentifier Collision - The Info.plist CFBundleIdentifier value 'mokumoku.RxWebKit' of 'MyApp.app/Frameworks/RxWebKit.framework' is already in use by another application.
```

I solved the problem by 
`INFOPLIST_FILE = Example/Info.plist;` 
with 
`INFOPLIST_FILE = RxWebKit/Info.plist;`for `Target: RxWebKit`

I will send a PullRequest containing those fix, if that's OK.

**Comment by freak4pc** [comment]
I think the problem is here:
https://github.com/RxSwiftCommunity/RxWebKit/blob/cf0b60ca35953e6f6891aa227b7c7eb954de8b2a/RxWebKit.xcodeproj/project.pbxproj#L600

The value shouldn't be `YES`, it should be `$(inherited)` probably. I don't want it as `NO` because that has its own repercussions.

I don't know enough about Carthage - but I'm not sure why the Bundle identifier would matter too much. If you have an idea of how to fix that, I'm all for a PR.

**Comment by sy-hash** [comment]
Thank you for your quick reply!
I just had sent PRs for each of these issues.

I would appreciate if you could review them.

**Comment by sy-hash** [comment]
This issue has been resolved in [v1.0.1].(https://github.com/RxSwiftCommunity/RxWebKit/releases/tag/1.0.1).
Thank you!
